bpf/node_config.h

```diff
@@ -128,7 +128,6 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define ENCRYPT_MAP test_cilium_encrypt_state
 #define TUNNEL_MAP test_cilium_tunnel_map
 #define VTEP_MAP test_cilium_vtep_map
-#define EP_POLICY_MAP test_cilium_ep_to_policy
```

Comment:
Let's remove `SOCK_OPS_MAP` on line 124 in `bpf/node_config.h` as well. It's now unused.

Reply:
Woah! I thought it was all about using `grep` to remove occurrences :sweat_smile:  but this one is hard to spot